src/modules/components/proposal/proposalActions/actions/proposalActionWithdrawToken/proposalActionWithdrawToken.tsx

```diff
@@ -0,0 +1,25 @@
+import { AssetTransfer } from '../../../../asset';
+import type { IProposalActionWithdrawToken } from '../../proposalActionsTypes';
+
+export interface IProposalActionWithdrawTokenProps {
+    /**
+     * Withdraw token action
+     */
+    action: IProposalActionWithdrawToken;
+}
+
+export const ProposalActionWithdrawToken: React.FC<IProposalActionWithdrawTokenProps> = (props) => {
+    const { action } = props;
+    return (
+        <AssetTransfer
+            sender={action.sender}
+            recipient={action.receiver}
+            assetName={action.token.name}
+            assetAmount={action.amount}
+            assetSymbol={action.token.symbol}
+            assetIconSrc={action.token.logo}
+            // TODO: Amend AssetTransfer hash req for proposal usage where no Tx exists yet, should disable asset link out
```

Comment:
Could you create a Jira ticket and link it here?
```suggestion
            // TODO: Make hash property on AssetTransfer optional (APP-XXXX)
```

Reply:
Will do! We were requiring it before because it would break the link creator without it, but can be handled.